pkg/terraform/stages/compat/stage.go

```diff
@@ -104,15 +102,6 @@ func extractHostAddresses(config *types.InstallConfig, tfstate *terraform.State)
 			logrus.Error(err)
 		}
 
-	case vspheretypes.Name:
-		bootstrap, err = gathervsphere.BootstrapIP(config, tfstate)
-		if err != nil {
-			return
-		}
-		masters, err = gathervsphere.ControlPlaneIPs(config, tfstate)
-		if err != nil {
-			logrus.Error(err)
-		}
```

Comment:
> The plan is to handle the gather with the output variables like the other platform stages as we could not reproduce issue with it at the moment. This should give us the opportunity to have other people than me test these changes as well.
> 
> However, if there is any issue, we would want to implement a custom host address extraction that should talk directly to vcenter instead 

This _already_ exists why not just use it?

> of looking up the tfstate files, because my understanding is that the tfstate file content should reflect the output variables for the bootstrap ip and control node ips

@e-tienne if you are not waiting for the ip address from vmware tools in terraform we cannot guarantee that the ip addresses will be available.

Reply:
> > The plan is to handle the gather with the output variables like the other platform stages as we could not reproduce issue with it at the moment. This should give us the opportunity to have other people than me test these changes as well.
> > However, if there is any issue, we would want to implement a custom host address extraction that should talk directly to vcenter instead
> 
> This _already_ exists why not just use it?
> 
> > of looking up the tfstate files, because my understanding is that the tfstate file content should reflect the output variables for the bootstrap ip and control node ips
> 
> @e-tienne if you are not waiting for the ip address from vmware tools in terraform we cannot guarantee that the ip addresses will be available.

Clarification: 
- we will use the IP gather based on the moid for vsphere that already exists.
- the configuration for waiting for the IP address from vmware tools was removed after my last comment, because we plan to reliably get the machine IP addresses from vsphere.